_posts/0000-01-29-githubteacher.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Hollenberry's Slide"
 ---
 
 ![fresh-prince](https://cloud.githubusercontent.com/assets/16547949/25400882/f44a7464-29c1-11e7-8f45-327d67cb961a.jpg)
 {: .center}
 
-Caption here
+I wasn't born and raised in West Philadelphia
```

Comment:
if you change your image, your caption might need to be changed as well

Reply:
💯